freqtrade/freqai/prediction_models/BaseRegressionModel.py

```diff
@@ -0,0 +1,112 @@
+import logging
+from typing import Tuple
+
+from pandas import DataFrame
+
+from freqtrade.freqai.data_kitchen import FreqaiDataKitchen
+from freqtrade.freqai.freqai_interface import IFreqaiModel
+
+
+logger = logging.getLogger(__name__)
+
+
+class BaseRegressionModel(IFreqaiModel):
+    """
+    Base class for regression type models (e.g. Catboost, LightGBM, XGboost etc.).
+    User *must* inherit from this class and set fit() and predict(). See example scripts
+    such as prediction_models/CatboostPredictionModel.py for guidance.
+    """
+
+    def return_values(self, dataframe: DataFrame, dk: FreqaiDataKitchen) -> DataFrame:
+        """
+        User uses this function to add any additional return values to the dataframe.
+        e.g.
+        dataframe['volatility'] = dk.volatility_values
+        """
+
+        return dataframe
+
+    def train(
+        self, unfiltered_dataframe: DataFrame, pair: str, dk: FreqaiDataKitchen
+    ) -> Tuple[DataFrame, DataFrame]:
```

Comment:
The return hint is clearly wrong here (we're not returning a tuple).
That obviously makes the docstring wrong as well ... (also applies to BaseTensor model)